### PR TITLE
Done

### PR DESCRIPTION
**Pull request by mliljenberg**
Here is the Image Click elements that was shown during presentation. Should be working fine.

### CODE REVIEW

**Comment by nwalters512** [comment]
Please fix the python linter errors reported by Travis! You can run `npm run lint-python` in the repo root to see the errors locally.

File: exampleCourse/questions/ImageClick/info.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "uuid": "2ad15b73-3192-4cde-89f6-35e2b8875caf",
+    "title": "Select a value in a img",
+    "topic": "Algebra",
+    "tags": ["code", "v3"],
```
Comment (nwalters512) [inline exampleCourse/questions/ImageClick/info.json]: Don't need the `code` tag.

File: exampleCourse/questions/ImageClick/question.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<pl_question_panel>
+    <p>Select the point in the graph where U'(x) = 0 </p>
```
Comment (nwalters512) [inline exampleCourse/questions/ImageClick/question.html]: Use $$U\prime(x)=0$$ for nicely-formatted math :)

File: question-servers/freeform.js [hunk 3]
```diff
@@ -57,6 +57,7 @@ module.exports = {
             (callback) => {
                 // Read all files in the given path
                 fs.readdir(sourceDir, (err, files) => {
+
```
Comment (nwalters512) [inline question-servers/freeform.js]: Why the newline?

File: exampleCourse/questions/addNumbers/question.html [hunk 4]
```diff
@@ -3,4 +3,4 @@
   <p> What is the sum $c = a + b$?
 </pl_question_panel>
 
-<p>$c = $ <pl_number_input answers_name="c" comparison="sigfig" digits="3" />
+<p><pl_number_input answers_name="c" comparison="sigfig" digits="3" label="$c=$"/></p>
```
Comment (nwalters512) [inline exampleCourse/questions/addNumbers/question.html]: Probably doesn't belong in this changeset.

File: elements/pl_img_click/pl_img_click.py [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+
+        # Get test_height (optional)
+        test_height = pl.get_string_attrib(element, 'test_height', None)
+
+         # Get show_coordinates (optional)
+        show_coordinates = pl.get_string_attrib(element, 'show_coordinates', None)
+
+        # Create and return html
+        html_params = {'question': True, 'src': file_url, 'scale': scale,'answer_scale': answer_scale, 'test_x':test_x, 'test_y':test_y, 'test_width':test_width, 'test_height':test_height, 'show_coordinates':show_coordinates }
+        with open('pl_img_click.mustache', 'r') as f:
```
Comment (nwalters512) [inline elements/pl_img_click/pl_img_click.py]: You should change all calls to `open` to specify UTF-8 encoding, as in #944

**Comment by mwest1066** [comment]
Closing out this old PR.

**Pull request closed without merging**